packages/testlab/test/unit/test-sandbox.unit.ts

```diff
@@ -1,34 +0,0 @@
-// Copyright IBM Corp. 2017,2018. All Rights Reserved.
-// Node module: @loopback/testlab
-// This file is licensed under the MIT License.
-// License text available at https://opensource.org/licenses/MIT
-
-import {TestSandbox, expect, sinon} from '../..';
-import * as fs from 'fs';
-import {resolve} from 'path';
-
-describe('TestSandbox unit tests', () => {
-  const sandboxPath = resolve(__dirname, 'sandbox');
-  let sandbox: TestSandbox;
-  let mkdirSyncSpy: sinon.SinonSpy;
-
-  beforeEach(createSpies);
-  beforeEach(createSandbox);
-  afterEach(restoreSpies);
-
-  it('created a sandbox using mkdirSync', () => {
```

Comment:
@virkt25 why have you removed this test file? Do we have any other tests to verify that Sanbox works as intended?

Reply:
Never mind, I found the integration test file that's still around: [packages/testlab/test/integration/test-sandbox.integration.ts](https://github.com/strongloop/loopback-next/blob/76a7f56cd1166d4407f2a50b188db91a74f7deae/packages/testlab/test/integration/test-sandbox.integration.ts)

Reply:
Yea. I didn’t see much value in checking if an underlying method was called when a TestSandbox function was called since the test becomes tightly coupled with the implementation. I believe the integration tests validate this class the best.